cert/README.md

```diff
@@ -35,6 +35,8 @@ cert
 
 ###### Automatically create and maintain iOS code signing certificates.
 
+Cert is focused exclusively on code signing. You can create a new code signing identity for a certain environment (dev, distribution) and check for any existing valid certificates installed locally. 
```

Comment:
Something sounds a bit weird about this language. Maybe:

"You can create new code signing identities for different environments (development and distribution) and use any existing valid certificates installed locally."